components/LiteMintNFT.vue

```diff
@@ -111,6 +122,19 @@ const formError = computed(() => {
 })
 
 const isFormValid = computed(() => !formError.value?.length)
+const bookName = computed(() => localISCNData.value?.contentMetadata?.name || '')
+const imagePreviewUrl = computed(() => {
+  const { ARWEAVE_ENDPOINT } = useRuntimeConfig().public
+  if (!formState.imageUrl) {
+    return null
+  }
+
+  const url = formState.imageUrl
+    .replace('ar://', `${ARWEAVE_ENDPOINT}/`)
+    .replace('ipfs://', 'https://ipfs.io/ipfs/')
+
+  return (url.startsWith('http://') || url.startsWith('https://')) ? url : null
```

Comment:
Can we reuse this https://github.com/likecoin/nft-book-press/blob/6d6f67859c4d11124d04efae753db90b646daff9/utils/index.ts#L34-L38